File: No te limites/No te limites/Event.swift
//
//  Event.swift
//  No te limites
//
//  Created by Griselda Epifanio on 12/8/16.
//  Copyright © 2016 Jose Epifanio. All rights reserved.
//

import Foundation

class Event
{
    private var date:String = ""
    private var name:String = ""
    private var place:String = ""
    private var people:String = ""
    
    //Seters
    public func setDate(date: String) {
        self.date = date
    }
    
    public func setName(name: String) {
        self.name = name
    }
    
    public func setPlace(place: String) {
        self.place = place
    }
    
    public func setPeople(people: String) {
        self.people = people
    }
    
    //Geters
    public func getDate()-> String {
        return self.date
    }
    
    public func setName()-> String {
        return self.name
    }
    
    public func getPlace()-> String {
        return self.place
    }
    
    public func setPeople()-> String {
        return self.people
    }
    
}

File: Android/Notelimites/app/src/main/java/com/burocreativo/notelimites/io/models/Event.java
package com.burocreativo.notelimites.io.models;

/**
 * Created by Juan C. Acosta on 9/3/2016.

 * juancacosta25@gmail.com.com
 */
public class Event {

    private String date;
    private String name;
    private String place;
    private String people;

    public Event(String date, String name, String place, String people) {
        this.date = date;
        this.name = name;
        this.place = place;
        this.people = people;
    }

    public Event() {
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPlace() {
        return place;
    }

    public void setPlace(String place) {
        this.place = place;
    }

    public String getPeople() {
        return people;
    }

    public void setPeople(String people) {
        this.people = people;
    }
}
